source/tester/tester-tests.adb

```diff
@@ -63,6 +63,11 @@ package body Tester.Tests is
    function Is_Has_Pattern (List : GNATCOLL.JSON.JSON_Array) return Boolean;
    --  Check if List in form of ["<HAS>", item1, item2, ...]
 
+   function Is_Does_Not_Have_Pattern
+     (List : GNATCOLL.JSON.JSON_Array)
+      return Boolean;
+   --  Check if List in form of ["<DOES_NOT_HAVE>", item1, item2, ...]
```

Comment:
Could you add the description of "<DOES_NOT_HAVE>" to testsuite/ada_lsp/README.md?

Reply:
Agree.
Added: https://github.com/AdaCore/ada_language_server/pull/847/commits/8db744a58702099d506a192cc8ddd6eea289ed3a#diff-137ea228c3dbbb82abc5fd889e19bf873dac27ffff4380adc10b11a004dd9754R95